components/greaselion/browser/greaselion_download_service.cc

```diff
@@ -17,6 +17,7 @@
 #include "base/strings/utf_string_conversions.h"
 #include "base/task/post_task.h"
 #include "base/task_runner_util.h"
+#include "base/threading/platform_thread.h"
```

Comment:
what is this header for? I'm not seeing anything new that was added, is it a mistake or for something that was already in the code?

Reply:
Oops, that was added while I was debugging with `Sleep` calls. I'll remove it.